@navikt/core/react/src/list/List.types.ts

```diff
@@ -7,18 +7,18 @@ export interface ListProps extends React.HTMLAttributes<HTMLDivElement> {
   as?: "ul" | "ol";
   /**
    * List heading title.
-   * @deprecated Prop will be removed in future versions.
+   * @deprecated Use a "plain" <Heading> instead of `title`, Composition of smaller components preferred over smarter components.
    */
   title?: string;
   /**
    * List heading description.
-   * @deprecated Prop will be removed in future versions.
+   * @deprecated Use a "plain" <BodyShort> instead of `description`, Composition of smaller components preferred over smarter components.
    */
   description?: string;
   /**
    * Allows setting a different HTML h-tag.
    * @default "h3"
-   * @deprecated Prop will be removed in future versions.
+   * @deprecated The `title` prop above (which we decided to call title, but refer to as a heading here?!) affected by this heading tag is being deprecated, so this one follows suit.
```

Comment:
is this too self-deprecating for documentation? 😂 (I think it's funny to own up / humanize "mistakes").

Reply:
Can be fun sometimes, but here i think it might be a little out of context since we are referencing a different change. User also wont have the context of "above" in `The `title` prop  above` here.